blosc2/SChunk.py

```diff
@@ -128,6 +138,32 @@ def __init__(self, chunksize=None, data=None, **kwargs):
 
         super(SChunk, self).__init__(schunk=sc, chunksize=chunksize, data=data, **kwargs)
         self.vlmeta = vlmeta(super(SChunk, self).c_schunk, self.urlpath, self.mode)
+        self.cparams_ = super(SChunk, self).get_cparams()
```

Comment:
For private params, better use the prefix version of the underscore.  E.g. `self._cparams`.  Besides of provide a hint that this is variable, autocompletion would not confuse the user.

Reply:
Fixed in 8fe520bc3b8fdaa36991a8037bec4163a263596e.